.travis.yml

```diff
@@ -61,16 +61,37 @@ matrix:
     - $TRAVIS_BUILD_DIR/ci/travis_install_clang_tools.sh
     script:
     - $TRAVIS_BUILD_DIR/ci/travis_lint.sh
-  - name: "C++ unit tests, code coverage with gcc 4.8"
+  - name: "C++ unit tests w/ Valgrind, gcc 4.8"
```

Comment:
Since code coverage isn't running in this build, we could use clang-6 here if we want (it blows up the coverage job for some reason)

Reply:
Is clang better for our use? I'm not sure it makes much of a difference.

Reply:
It will turn up a lot more compiler warnings. In the past some missing `override` passed through CI undetected